homeassistant/helpers/script.py

```diff
@@ -677,12 +677,18 @@ async def _async_call_service_step(self):
                 **params,
                 blocking=True,
                 context=self._context,
-                limit=limit,
             )
         )
         if limit is not None:
```

Comment:
we can remove this whole if-clause and remove all handling of `limit` in this block.

Reply:
Removed all limits.

Reply:
Removed.